src/styles/AddCar.css

```diff
@@ -0,0 +1,104 @@
+.form-container {
```

Comment:
- [ ] In the mobile view, if I open the nav bar to go to "add car":

![image](https://github.com/Chandan-devs-tech/drive-easy-frontend/assets/57421823/8934089c-8d49-49f2-8c99-7fa53b84a014)

1. The navbar won't close on selecting the link.  I have to close the nav bar manually which is inconsistent with the rest of the links.

2. The content of "add car" is overflowed by the content of "home" if I'm coming from there:

![image](https://github.com/Chandan-devs-tech/drive-easy-frontend/assets/57421823/1055324e-569e-4e58-bfb0-260041565699)

Kindly, fix these styling issues so your mobile view works as intended. 👍